### ISSUE TITLE
notejs issues 

### ISSUE THREAD

**Issue by DanielAbsi** [apg-intel/ipv6tools#18]
HI I have an error with localhost and note js /cmder:

npm ERR! Windows_NT 10.0.17134
npm ERR! argv "D:\\Program Files\\nodejs\\node.exe" "D:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "setup"
npm ERR! node v6.11.0
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! ipv6tools@1.0.0 setup: `sudo pip install -r requirements.txt && npm install && npm run build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ipv6tools@1.0.0 setup script 'sudo pip install -r requirements.txt && npm install && npm run build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the ipv6tools package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     sudo pip install -r requirements.txt && npm install && npm run build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs ipv6tools
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls ipv6tools
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:

**Comment by joshporter1** [comment]
Could you provide more details about your environment? OS, node version, etc. Thanks!

**Comment by vp93** [comment]
It seems like you have outdated node and npm versions ( v6.11.0 & v3.10.10 respectively )
Some node modules might fail to install as there have been breaking changes in the latest versions : 
Download the latest build of node from their official site [here](https://nodejs.org/en/)

This should automatically update the node and npm to the latest versions. 
Re-try installing after that.

**Comment by joshporter1** [comment]
Correct, it seems you're using an older version of node, possibly pre-installed with the OS. Upgrading should solve this. 

Alternatively, if you are looking to use the tool and not develop features, you can just run the following.
```bash
sudo pip install -r requirements.txt
sudo python app.py
```

This runs the tool using the prebuilt assets.
